docs/scenarios_authoring/AggregatesInTimeWindows.md

```diff
@@ -77,15 +70,15 @@ The result of the `groupBy` expression must be of type String.
 
 *result is held in the variable configured in the `output` field.
 
-## Common behaviour
-Components which produce aggregates in time windows process multiple events; a question may arise about the contents of the variables when the window is closed.
-Presence of variables defined before aggregation (e.g. `#input`, `#inputMeta`) depends on configuration of the `emitWhenEventLeft` parameter. `#key` parameter is added for every aggregation type and holds a key which is used in `groupBy` 
+## Common behavior
+
+The sliding, tumbling and session window components differ among themselves not only in the way the time window is defined. A second key aspect which differentiates these components is how they `transform' input events into aggregate. This transformation can be either creation of a new (aggregate) event or an enrichment of incoming events with the value of the aggregate.  
+
+When the time window is closed, tumbling and session window nodes generate a new event containing an aggregate. Its timestamp is equal to the time of the timer that generated it, not system time of the moment when it happened. In other words the timestamp of the newly generated event which contains the aggregate will continue to use the notion of time used by events which the aggregate window saw. These nodes 'terminate' the events which entered them; consequently the #input and #inputMeta variables will not be available downstream. The sliding-window behaves in the same way in its **non-default** configuration when parameter `emitWhenEventLeft` is set to `true`. 
+
+A sliding-window in its default configuration (`emitWhenEventLeft` is set to `false`) enriches the incoming event with the value of the aggregate. Because the original events are still available downstream, the #input and #inputMeta variables will be available 'downstream'.
```

Comment:
```suggestion
A sliding-window in its default configuration (`emitWhenEventLeft` is set to `false`) enriches the incoming event with the value of the aggregate. Because the original events are still available downstream, the `#input` and `#inputMeta` variables will be available 'downstream'.
```

Reply:
I would move this paragraph above the one about tumbling/session. This mode (sliding, without emitWhenEventLeft) is used often and is IMHO simpler conceptually